src/main/java/seedu/jarvis/model/ModelManager.java

```diff
@@ -423,11 +423,11 @@ public void updateMasteryChecksList(Predicate<MasteryCheck> predicate) {
         filteredMasteryChecks.setPredicate(predicate);
     }
 
-    @Override
+    /*
```

Comment:
Do remember to remove the comment once we are certain that we dont need this method :)